plan/physical_plan_builder.go

```diff
@@ -828,7 +823,7 @@ func (p *Aggregation) convert2PhysicalPlanHash() (*physicalPlanInfo, error) {
 		}
 	}
 	if !distinct {
-		if x, ok := childInfo.p.(physicalDistSQLPlan); ok {
+		if x, ok := childInfo.p.(physicalDistSQLPlan); ok && !x.isMemDB() {
```

Comment:
@idlesummerbreeze 
copied from `expression/aggregate.go:113`
```
const (
	// CompleteMode function accepts origin data.
	CompleteMode AggFunctionMode = iota
	// FinalMode function accepts partial data.
	FinalMode
)
```
TiKV regions calcuate partial aggregate results, we should use them to calculate the final aggregate result.